tests/msc3902/federation_room_join_partial_state_test.go

```diff
@@ -376,7 +376,7 @@ func TestPartialStateJoin(t *testing.T) {
 	// resync completes. This test does exactly that.
 	t.Run("EagerLongPollingSyncWokenWhenResyncCompletes", func(t *testing.T) {
 		alice := deployment.Register(t, "hs1", helpers.RegistrationOpts{
-			Localpart: "t1alice_long_polling",
+			LocalpartSuffix: "t1alice_long_polling",
```

Comment:
Aside: can you remember why we ended up making all these different users in the faster joins tests?

Reply:
No clue. Probably because they were parallel at some point, using the same HS deployment?

Reply:
Sounds plausible. I vaguely remember that we added them to make things less flakey.